pkg/features/list.go

```diff
@@ -130,7 +130,7 @@ var (
 	LLMRiskRecommendation = registerFeature("Intgrate with LLM for risk recommendations", "ROX_LLM_RISK_RECOMMENDATION")
 
 	// Locks process baselines when their deployments leave the observation period
-	AutoLockProcessBaselines = registerFeature("Locks process baselines when their deployments leave the observation period", "ROX_AUTOLOCK_PROCESS_BASELINES")
+	AutoLockProcessBaselines = registerFeature("Locks process baselines when their deployments leave the observation period", "ROX_AUTO_LOCK_PROCESS_BASELINES", enabled)
```

Comment:
Two other occurrences to replace, please:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/types/featureFlag.ts#L7
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Clusters/DynamicConfigurationForm.tsx#L41